### ISSUE TITLE
Build result when @conkeyref attribute to reference external content is possibly incorrect for the html5 transform

### ISSUE THREAD

**Issue by faashford** [dita-ot/dita-ot#2983]
## Overview

DITA builds using the html5 transform do not handle @conkeyref attribute references outside of the directory hierarchy underneath the map referenced by args.input. The path to this file is miscalculated. This problem has few follow-on consequences:
        - Content in t-prompt.dita is not pulled in to the output.
        - @conkeyref references for shared content in a file in the same directory as the map is not pulled into the output.
        - Paths to the CSS file are miscalculated.

Below is an overview of the content source setup. File src/shared_content/t-prompt.dita will not be seen if this is built:

```
src/
    shared_content/ 
        t-prompt.dita
    topicset/
        m-map.ditamap
        c-top_doc.dita
        t-conkeyref-test.dita
        to-data.dita
```

In the example, m-map.ditamap has keyrefs:

    <keyref keys="prompts" href="../shared_content/t-prompt.dita" processing-role="resource-only"/>
    <keyref keys="ts" href="to-data.dita" processing-role="resource-only"/>

The file t-conkeyref-test.dita has a @conkeyref set like this:

    <systemoutput conkeyref="prompts/python"/>

The result is that the path to shared content calculated by the transform appears to be incorrect.  Impacts include path to to-data.dita being miscalculated, path to CSS file being miscalculated as well as path to t-prompt.dita being miscalculated.

## Steps to Replicate

1.  Unzip the attached file.
2.  Start a Windows command shell with the appropriate path setup.
3.  Navigate to Example/DPAD_component.
4.  Build the content by running the render.cmd script.
5.  Find the errors in output.log.
6.  Look at the source code for t-conkeyref-test.html, noting the path descrepencies to the CSS file.

## Actual Results

    .   Titles are missing
    .   Items referenced through @conkeyref are missing
    .   CSS path in HTML metadata is incorrect.
    
## Expected Results

To my wave of thinking, @conkeyrefs would be handled as they are in 1.x.

## Platform and Build

Windows 7
DITA 3.0.3

## Additional Platforms and Builds

I have seen this with all of the 2.x versions as well, also with Windows 7.

## Additional Information

I have tried xhtml and eclipsehelp both of which have the same problem as the html5 transform.

Notably this problem does not occur with the pdf transform.

**Comment by faashford** [comment]
[Example.zip](https://github.com/dita-ot/dita-ot/files/2059474/Example.zip)

**Comment by infotexture** [comment]
Fixed for 3.1.1 w/ PR #3014.
